src/test/java/teammates/test/cases/action/InstructorEditStudentFeedbackPageActionTest.java

```diff
@@ -195,6 +196,20 @@ protected InstructorEditStudentFeedbackPageAction getAction(String... params) {
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+
+        StudentAttributes student = typicalBundle.students.get("student1InCourse1");
+
+        String feedbackSessionName = "First feedback session";
+        String courseId = student.course;
+        String moderatedStudentEmail = student.email;
+
+        String[] submissionParams = new String[]{
+                Const.ParamsNames.COURSE_ID, courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON, moderatedStudentEmail
+        };
+
+        verifyOnlyInstructorsOfTheSameCourseCanAccess(submissionParams);
+        verifyUnaccessibleWithoutModifySessionPrivilege(submissionParams);
```

Comment:
`verifyUnaccessibleWithoutModifySessionCommentInSectionsPrivilege` is more appropriate here.